pyinterprod/cli.py

```diff
@@ -389,24 +389,14 @@ def run_member_db_update():
     if member_dbs or feature_dbs or site_dbs:
         tasks += [
             Task(
-                fn=interpro.iprscan.import_tables,
+                fn=interpro.iprscan.update_tables,
                 args=(ora_iprscan_uri, "matches"),
                 kwargs=dict(databases=member_dbs + feature_dbs + site_dbs,
                             force=True,
                             threads=8),
-                name="import-ipm-matches",
-                scheduler=dict(type=scheduler, queue=queue, mem=100, hours=24)
-            ),
-            Task(
-                fn=interpro.iprscan.update_partitions,
-                args=(ora_iprscan_uri, "matches"),
-                kwargs=dict(databases=member_dbs + feature_dbs + site_dbs,
-                            force=True,
-                            threads=4),
                 name="update-ipm-matches",
-                scheduler=dict(type=scheduler, queue=queue, mem=100, hours=24),
-                requires=["import-ipm-matches"]
-            ),
+                scheduler=dict(type=scheduler, queue=queue, mem=100, hours=24)
```

Comment:
24 hours might be enough, but increase time limit to be on the safe side.

```suggestion
                scheduler=dict(type=scheduler, queue=queue, mem=100, hours=48)
```